staking/contracts/contracts.py

```diff
@@ -275,6 +302,9 @@ def reward(
     """Primarily used to supply the initial rewards for the staking contract,
```

Comment:
In the Readme, there is a comment:
> When deploying a new pool it is assumed that the full total supply of the reward asset is being deposited into the contract.
If siphoning off some of the reward asset is a security violation, then consider atomically creating the reward asset when supplying it to the app. This should be possible with an inner transaction.